src/main/java/com/example/gradecalculator/web/controller/FormController.java

```diff
@@ -1,34 +1,37 @@
 package com.example.gradecalculator.web.controller;
-
-import com.example.gradecalculator.entities.UserType;
-import com.example.gradecalculator.repository.UserTypeRepository;
-import org.apache.catalina.User;
+import com.example.gradecalculator.entities.AllUsers;
+import com.example.gradecalculator.repository.AllUsersRepository;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.autoconfigure.security.SecurityProperties;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.ResponseBody;
 
 @Controller
+@SpringBootApplication
```

Comment:
a controller should never be the entry point of your application

Reply:
yup, fixed